tietokanta/src/main/resources/db/migration/V1_830__.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE paallystysmassa
+(
+    id                SERIAL PRIMARY KEY,
+    urakka            INTEGER REFERENCES urakka (id),
+
+    massatyyppitunnus TEXT,
+    raekoko           INTEGER,
```

Comment:
tässä taulun määrityksessä pistää silmään nyt jälkikäteen, että pitää analysoida vielä, mitkä kentistä olisi syytä määrittää NOT NULL